### PR TITLE
fix: change to select colors via token

### LINKED ISSUE #2246: Let's unify color consistency with token colors
### Main idea

In dark mode, the button colors and resource monitor colors are not unified and look incomplete.Let's improve this.

image

### Alternative ideas

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by ironAiken2**
### Feature
- set mwc button bg and progress bar color via token. For dark mode, I used colorPrimary. 
- The colors will be changed depending on the theme.
- Changed the hard-coded color using antd token.

### Reflected request changes
- set BAI app download button to non-primary and non-shadow, and fixed width between button and selector. I added icons to help users understand.
- rollback manager info tag color in information page. 
- delete colorDisabled in theme.json.

image

**Checklist:** (if applicable)

- [ ] Mention to the original issue
- [ ] Documentation
- [ ] Minium required manager version
- [ ] Specific setting for review (eg., KB link, endpoint or how to setup)
- [ ] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

**Comment by yomybaby** [comment]
image

Let's change this download button to non-primary. It's not the primary action, and it's not on the same level as the "start" button.

**Comment by yomybaby** [comment]
image
image

The color of the progress bar in dark mode is brighter than in light mode. Could you improve this?

**Comment by yomybaby** [comment]
image

Please change the color of the icon in "지역" column.

**Comment by yomybaby** [comment]
Before
image

After this PR
image

The "before" is better.

File: resources/theme.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
       "colorSuccess": "#37B076",
       "colorBorder": "#DEE1E7",
       "colorBorderSecondary": "#E1E1E2",
-      "colorWarning": "#f1a239"
+      "colorWarning": "#f1a239",
+      "colorDisabled": "rgba(255, 255, 255, 0.04)"
```
Comment (yomybaby) [inline resources/theme.json]: Why did you set the `colorDisabled` color?

**Review by yomybaby — CHANGES_REQUESTED** [review]

**Comment by ironAiken2** [comment]
@yomybaby 
I reflect changes requested. 
However, the icons representing `regions` on the `agent page` are saved as pngs, which prevents me from changing their color. 

In a previous commit, I had a misunderstanding about the theme.json file. now it's fixed

File: src/components/backend-ai-general-styles.ts [hunk 2]
```diff
@@ -129,11 +126,14 @@ export const BackendAiStyles = [
       --general-progress-bar-bg: var(--token-colorBorderSecondary, #e8e8e8);
       --general-progress-bar-reserved: linear-gradient(
         to left,
-        #722cd7,
-        #5c7cfa
+        var(--token-blue-8) 0%,
```
Comment (ironAiken2) [inline src/components/backend-ai-general-styles.ts]: @yomybaby 
This is the part that controls the gradient color of the progress bar.

**Review by yomybaby — APPROVED** [review]
LGTM!

File: src/components/backend-ai-general-styles.ts (hunk 2 above)
Comment (yomybaby) [reply src/components/backend-ai-general-styles.ts]: I pushed a commit to remove the gradient color of the progress bar.

**Pull request merged**